src/main/java/athena/timetable/Timetable.java

```diff
@@ -81,18 +90,216 @@ private void populateTimetable() {
         }
     }
 
+    /**
+     * Generates the timetable header containing hour marks.
+     * For example, +------08------09------10------11------+
+     *
+     * @param startHour The starting hour in 24-hour representation.
+     * @param endHour   The ending hour in 24-hour representation.
+     * @return A string containing the timetable header with hour marks.
+     */
+    private String drawTimetableTimeHeader(int startHour, int endHour) {
+        String header = "";
+
+        header += "+" + "-------";
```

Comment:
maybe can use the string "+-------"?

Reply:
Oh I was planning to refactor `-------` into a constant. Will do that tomorrow.